probe-rs-cli-util/src/logging.rs

```diff
@@ -100,7 +146,38 @@ pub fn init(level: Option<Level>) {
         Ok(())
     });
 
-    builder.init();
+    // Sentry logging (all log levels except tracing (to not clog the server disk & internet sink)).
+
+    let mut sentry = Builder::new();
+
+    // Always use the Debug log level.
+    sentry.filter_level(LevelFilter::Debug);
+
+    // Define our custom log format.
+    sentry.format(move |_f, record| {
+        let mut log_guard = LOG.write().unwrap();
+        log_guard.push(Breadcrumb {
+            level: match record.level() {
+                Level::Error => sentry::Level::Error,
+                Level::Warn => sentry::Level::Warning,
+                Level::Info => sentry::Level::Info,
+                Level::Debug => sentry::Level::Debug,
+                Level::Trace => sentry::Level::Debug,
```

Comment:
Shouldn't this be `sentry::Level::Trace`?

Reply:
Sadly, Sentry has no trace level :(

Reply:
Please add a comment indicating this, so that it is clear for future readers.